src/Components/HeroSection/HeroSection.scss

```diff
@@ -81,8 +82,8 @@
 }
 
 .hero-image {
-	width: 687px;
-	height: 657px;
+	width: 680px;
+	height: 650px;
```

Comment:
- [ ] I think you can remove this width and height and apply a 100% width for the image element.

Reply:
Fixed